README.md

```diff
@@ -25,7 +25,7 @@ Core features include:
 <!-- toc -->
 
 - [Talawa Components](#talawa-components)
-- [Documentation](#documentation)
+- [Documentation -](#documentation--)
```

Comment:
Undo this.

Reply:
@noman2002 I have reverted the changes; please take a look at them.